src/test/java/seedu/address/logic/commands/SplitCommandTest.java

```diff
@@ -0,0 +1,125 @@
+package seedu.address.logic.commands;
```

Comment:
Can consider testing a case where some value is split such that the rounded values that do not sum up to the original value due to rounding can be added back together to the original number.

For example: $1 split among 3 is $0.33, but when you sum up the total owed etc. it becomes $0.99, when it should be $1.